crates/xilem_masonry/src/view/flex.rs

```diff
@@ -1,11 +1,14 @@
 use std::marker::PhantomData;
 
 use masonry::{
-    widget::{self, Axis, WidgetMut},
+    widget::{self, Axis, FlexParams, WidgetMut},
     Widget, WidgetPod,
 };
 
-use crate::{ChangeFlags, ElementSplice, MasonryView, VecSplice, ViewSequence};
+use crate::{
+    sequence::SequenceCompatible, ChangeFlags, ElementSplice, MasonryView, VecSplice, View,
+    ViewElement, ViewSequence,
+};
 
 // TODO: Allow configuring flex properties. I think this actually needs its own view trait?
 pub fn flex<VT, Marker>(sequence: VT) -> Flex<VT, Marker> {
```

Comment:
(not directly related to that PR)

I'm not sure whether we actually should add the type bounds here (generally for these "component" functions).

I think for one it helps a little bit with error messages, and since I think it should only be used inside a `View` context, it also shows, what is actually allowed here as sequence. (But well, unfortunately it adds a lot of generic parameters obviously)